src/antsibull/data/ansible-setup_py.j2

```diff
@@ -174,17 +178,45 @@ setup(
     },
     license='GPLv3+',
     python_requires='{{ python_requires }}',
+{%- if version.major >= 8 %}
+    packages=find_namespace_packages(
+        '.',
+        include=[
+            'ansible_collections*',
+        ],
+        exclude=[
+{%-   for collection in collection_names %}
+            'ansible_collections.{{ collection }}.tests.*',
+            'ansible_collections.{{ collection }}.tests',
+            'ansible_collections.{{ collection }}.docs.*',
+            'ansible_collections.{{ collection }}.docs',
+{%-   endfor %}
+        ],
+    ),
+    package_dir={'': '.'},
+    package_data={'': ['*', '*/*/*/**/.*']},
```

Comment:
What is `'*/*/*/**/.*'` trying to to do?

Reply:
It matches all dotfiles in subdirectories of collection roots, but none in collection roots (or further up the tree).